code/game/objects/items/stacks/stack.dm

```diff
@@ -391,11 +394,11 @@
 
 	return TRUE
 
-/obj/item/stack/proc/zero_amount()
+/// Returns TRUE if the item stack is the equivalent of a 0 amount item.
+/obj/item/stack/proc/is_zero_amount()
```

Comment:
Won't this cause cyborg stacks to get deleted in all these ?

Reply:
Yes it will. I've come up with another solution which is a bit dirty, but gets the job done.